.github/workflows/live-test.yml

```diff
@@ -0,0 +1,45 @@
+# This workflow is triggered by the user and runs live tests on the codebase.
+name: Live Test
+
+on:
+  workflow_dispatch:
```

Comment:
Didn't we decide to switch this trigger to be label based?

Reply:
I was just about to push that change :)